shapetree.ttl

```diff
@@ -0,0 +1,325 @@
+# It is primarily used for use cases where a ShapeTree must be described to
+# a user for purposes of authorization.
+#############################################################################
+:ShapeTreeDecoratorIndex 
+  a rdfs:Class ;
+  rdfs:comment "Provides an index of SKOS poly-hierarchies which can be used to describe a ShapeTree"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "ShapeTreeDecoratorIndex"@en .
+
+# Properties in domain of ShapeTreeDecoratorIndex
+#########################################################
+:hasSeries
+  a owl:ObjectProperty ;
+  rdfs:domain :ShapeTreeDecoratorIndex ;
+  rdfs:range :ShapeTreeDecoratorSeries ;
+  rdfs:comment "Links between a ShapeTreeDecoratorIndex and a ShapeTreeDecoratorSeries"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Has Series"@en .
+
+# ShapeTreeDecoratorSeries class
+#############################################################################
+# A ShapeTreeDecoratorSeries details a versioned set of ShapeTreeDecoratorHierarchy classes
+# Primary use is grouping TreeDecoratorHierarchy instances by a common discriminator
+# such as language or intended audience
+#############################################################################
+:ShapeTreeDecoratorSeries 
+  a rdfs:Class ;
+  rdfs:comment "Groups a set of ShapeTreeDecoratorHierarchy by a given discriminator"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "ShapeTreeDecoratorSeries"@en .
+
+#
+# Properties in domain of ShapeTreeDecoratorSeries
+#
+:hasHierarchy
+  a owl:ObjectProperty ;
+  rdfs:domain :ShapeTreeDecoratorSeries ;
+  rdfs:range :ShapeTreeDecoratorHierarchy ;
+  rdfs:comment "Links between a ShapeTreeDecoratorSeries and a ShapeTreeDecoratorHierarchy"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Has Hierarchy"@en .
+
+# ShapeTreeDecoratorHierarchy class
+#############################################################################
+# A ShapeTreeDecoratorHierarchy details a versioned SKOS poly-hierarchy that describes
+# a ShapeTree in a given language or format for an intended audience 
+# (decreased cognitivty ability, etc.)
+#############################################################################
+:ShapeTreeDecoratorHierarchy 
+  a rdfs:Class ;
+  rdfs:comment "Describes a SKOS poly-hierarchy for purproses of labeling a ShapeTree"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "ShapeTreeDecoratorHierarchy"@en .
+
+# Properties in domain of ShapeTreeDecoratorHierarchy
+#########################################################
+:hasVersion
+  a owl:DatatypeProperty ;
+  rdfs:domain :ShapeTreeDecoratorHierarchy ;
+  rdfs:range xsd:string ;
+  rdfs:comment "Describes the version of a ShapeTreeDecoratorHierarchy"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "Has Version"@en .
+
+:hasSHA3256Hash
+  a owl:DatatypeProperty ;
+  rdfs:domain :ShapeTreeDecoratorHierarchy ;
+  rdfs:range xsd:string ;
+  rdfs:comment "Describes the SHA3-256 of a ShapeTreeDecoratorHierarchy"@en ;
```

Comment:
Given some of the feedback we got on using SHA3 in ecosystem proposal would SHA-256 be sufficient?

Reply:
Updated.

Reply:
@joshdcollins Comment and label looks good but the actual property name is still `:hasSHA3256Hash` 🙂

Reply:
Updated for real.